packages/forms/src/UIForm/utils/properties.test.js

```diff
@@ -105,5 +113,26 @@ describe('Properties utils', () => {
 				},
 			});
 		});
+
+		it('should delete the value, creating nested objects', () => {
```

Comment:
Can you change the description please ? `creating nested objects` is not good here